src/components/cards/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import { cva, type VariantProps } from 'class-variance-authority'
+import { Card as BaseCard, CardTitle as BaseCardTitle, CardContent as BaseCardContent } from '../ui/card'
+import { cn } from '../../lib/utils'
+export { CardFooter, CardHeader, CardAction, CardDescription } from '../ui/card'
+
+const cardVariants = cva('rounded-sm', {
+  variants: {
+    variant: {
+      default: '',
+      layout: 'flex-1 gap-2 bg-background mb-4 md:mb-6',
+    },
+  },
+  defaultVariants: {
+    variant: 'default',
+  },
+})
+
+type CardProps = React.ComponentProps<typeof BaseCard> & VariantProps<typeof cardVariants>
+
+export const Card = ({ className, variant, ...props }: CardProps) => (
+  <BaseCard className={cn(cardVariants({ variant }), className)} {...props} />
+)
+
+type CardTitleProps = React.ComponentProps<typeof BaseCardTitle>
+
+export const CardTitle = ({ className, ...props }: CardTitleProps) => (
```

Comment:
The card "With action" story looks kinda broken: https://6d790ebb.oasis-ui.pages.dev/iframe?viewMode=docs&id=components-card--docs&globals=#with-action